src/deepsparse/transformers/pipelines/masking.py

```diff
@@ -0,0 +1,336 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# postprocessing adapted from huggingface/transformers
+
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+Pipeline implementation and pydantic models for masking transformers
+tasks
+"""
+from typing import Any, Dict, List, Optional, Tuple, Type, Union
+
+import numpy
+from pydantic import BaseModel, Field
+from transformers.file_utils import ExplicitEnum
+from transformers.tokenization_utils_base import PaddingStrategy, TruncationStrategy
+
+from deepsparse import Pipeline
+from deepsparse.transformers.pipelines import TransformersPipeline
+
+__all__ = [
+    "AggregationStrategy",
+    "MaskingInput",
+    "MaskingResult",
+    "MaskingOutput",
+    "MaskingPipeline",
+]
+
+
+class AggregationStrategy(ExplicitEnum):
+    """
+    Valid aggregation strategies for postprocessing in the MaskingPipeline
```

Comment:
I would love to here more explanation what each strategy represents.

Reply:
I have removed this functionality it is not required.